app/components/outside-click/component.js

```diff
@@ -0,0 +1,53 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+  classNames: ['outside-click'],
+  layout,
+  isOutside: false,
+  excludedClasses: [],
+
+  onOutsideClick() {},
+
+  init() {
+    this._super(...arguments);
+    this.handleDown = this.handleDown.bind(this);
+    this.handleUp = this.handleUp.bind(this);
```

Comment:
Save these to a separate variable (like `_boundHandleDown`) instead of overwriting the original name and leaving no direct reference to it.